CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing guide
+
+Contributions and feedback on your experience of using this software are welcome.
+
+This includes bug reports, feature requests, ideas, pull requests, and examples of how you have used this software.
+
+Please see the [Code of Conduct](CODE_OF_CONDUCT.md) and follow any templates configured in GitHub when reporting bugs, requesting enhancements, or contributing code.
+
+Please raise any significant new functionality or breaking change an issue for discussion before raising a Pull Request for it.
+
+## For contributors
+
+Anyone can be a contributor. Either you found a typo, or you have an awesome feature request you could implement, we encourage you to create a Pull Request.
+
+### Pull Requests
+
+- The latest changes are always in `main`, so please make your Pull Request against that branch.
+- Pull Requests should be raised for any change
+
+### Setting up local environment
+
+The local environment can be setup by doing the following.
+
+1. Clone the repository `$ git clone https://github.com/nextauthjs/docs.git`
+2. Install dependencies `$ cd docs && npm install`
+3. Start development server `$ npm start`
+
+## For maintainers
+
+### Recommended Scopes
+
+A typical conventional commit looks like this:
+
+```
+type(scope): title
+
+body
+```
+
+Scope is the part that will help grouping the various commit types in the release notes.
+
```

Comment:
```suggestion
```

I don't think this part is necessary, docs is not subject to semver, so I don't actually care about commit formats on the repo. I guess we can do it anyway to be nice, but not necessary to require it.

Reply:
Yeah good call :+1: